src/check.rs

```diff
@@ -28,7 +28,8 @@ pub fn set_cargo_config_env() -> anyhow::Result<()> {
     let cargo = env::var_os("CARGO");
     let mut child = Command::new(cargo.as_deref().unwrap_or_else(|| "cargo".as_ref()))
         .args(["config", "get", "-Zunstable-options", "--format=json"])
-        .stdout(Stdio::null())
+        .stdout(Stdio::piped())
```

Comment:
Use default for stdout

Reply:
wdym, `Stdio` doesn't implement `Default` trait, and not explicitly setting it would use `Stdio::inherit()` since we do `.spawn()` here;

It would default to `::piped()` if we did `.output()` instead of `.spawn()` though.
I can rewrite it to `.output()`, that would simplify code a bit.

Reply:
Sorry for confusion. First you thought I was wrong, then I thought you were wrong, then i thought i was wrong, then you thought i was wrong in thinking i was wrong. Should be right according to my initial assessment.

Reply:
Haha, yeah, recursive confusion :D